e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/editOnKeyPress.screenshots.ts

```diff
@@ -62,3 +65,55 @@ const DATA_GRID_SELECTOR = '#container';
     await makeRowsViewTemplatesAsync(DATA_GRID_SELECTOR);
   });
 });
+
+test('Focused cell should not flick', async (t) => {
```

Comment:
Let's add the ticket number to test case name
E.g:
```suggestion
test('Focused cell should not flick (T1206435)', async (t) => {
```